universal-application-tool-0.0.1/app/views/questiontypes/RepeaterQuestionRenderer.java

```diff
@@ -20,10 +32,14 @@ public RepeaterQuestionRenderer(ApplicantQuestion question) {
 
   @Override
   public Tag render() {
+    ContainerTag repeaterFields = div().withId(REPEATER_FIELDS_ID);
+    // TODO: add each answer as a repeaterField
```

Comment:
suggestion (non-blocking): Can we also add a TODO for making the placeholder name an admin-configurable field, with a link to an issue?

Reply:
Added a TODO for #859.

Reply:
Thanks!